### DISCUSSION TITLE
Marker pack for Bjora essence chests

### DISCUSSION THREAD

**Discussion by Harriss77** [blish-hud/Blish-HUD#702] [Requested Marker Packs]
I could use a path for just the Bjora essence chests.  I lead Drakkar and Storms of Winter for my guild, and would love a path for just those chests to fill the time between the events.  Thanks in advance!

**Comment by Metallis** [comment]
https://wiki.guildwars2.com/wiki/Essence_chest
Desired stops: medium and large chests

**Reply by Metallis** [reply]
MVP created, awaiting user trial before incorporation.

**Comment by Metallis** [comment]
This has been completed and integrated into https://github.com/Metallis/Metal-Marker-Myriad, permission granted to Harriss and ViP guild to use separately.
